sympy/matrices/tests/test_matrices.py

```diff
@@ -2276,6 +2276,11 @@ def test_GramSchmidt():
     assert GramSchmidt([Matrix([3, 1]), Matrix([2, 2])], True) == [
         Matrix([3*sqrt(10)/10, sqrt(10)/10]),
         Matrix([-sqrt(10)/10, 3*sqrt(10)/10])]
+    # Issue #9488
+    from sympy import FiniteSet
+    L = FiniteSet(Matrix([1]))
+    print(GramSchmidt(L))
```

Comment:
Imports should go at the top of the file.

There should be no print here.

I think it's better to include a link to the issue in a comment rather than just the issue number.

Reply:
Oka